### DISCUSSION TITLE
Plotting utility

### DISCUSSION THREAD

**Discussion by jklymak** [c-proof/pyglider#88] [General]
@callumrollo do you use our plotting at all?  I'm wondering if this should be taken out of pyglider, and we let users figure out their own plotting?

**Comment by callumrollo** [comment]
We use our own plotting routines. I think it would be better to keep pyglider focused on initial data processing. Other packages like GliderTools have some good inbuilt plotting along with glider data post-processing

**Reply by jklymak** [reply]
OK, we will move `website` and `plotting` out of here.  plotting is so idiosyncratic that the odds of anyone agreeing with how we do it are low.  We can always re-introduce.

**Reply by jklymak** [reply]
Mostly done in #89  - I kept a helper in "utils"
